src/fillTank.test.js

```diff
@@ -1,11 +1,213 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+  it('should not return anything', () => {
```

Comment:
. You can provide more descriptive test case descriptions to make it clear what each test is checking. 
```suggestion
  it('should not return anything when filling the tank', () => {
```